### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by jbaron-mx** [getcandy/candy-api#199]
Hey guys,

I wanted to check the test coverage on the project, however, I'm getting errors when trying to run the tests, not sure if I'm doing something wrong or the tests need work, hopefully, you can give me a hand.

## Expected Behavior
All tests should pass.

## Current Behavior
It is failing on test `Tests\Feature\Http\Controllers\Attributes\CategoryControllerTest::test_can_list_all_categories`

## Steps to Reproduce
1. Clone the repo `git clone https://github.com/getcandy/candy-api candy`
2. `composer install`
3. `./vendor/bin/phpunit`

I didn't change anything else. `phpunit.xml` looks configured to use SQLite as the testing database.
I'm using Laravel Homestead as my dev environment, latest version.

It seems to be an issue when migrating the tables.
PHPUnit output is as follows:
```
❯ ./vendor/bin/phpunit
PHPUnit 8.5.5 by Sebastian Bergmann and contributors.

E

Time: 529 ms, Memory: 28.00 MB

There was 1 error:

1) Tests\Feature\Http\Controllers\Attributes\CategoryControllerTest::test_can_list_all_categories
Illuminate\Database\QueryException: SQLSTATE[HY000]: General error: 1 no such table: languages (SQL: insert into "languages" ("lang", "iso", "name", "default", "updated_at", "created_at") values (en, gb, English, 1, 2020-05-24 17:37:54, 2020-05-24 17:37:54))
```

**Comment by alecritson** [comment]
Hey @jbaron-mx 

As a test, can you change the line in `phpunit.xml` as follows:

```
---
<env name="DB_CONNECTION" value="sqlite"/>
+++
<env name="DB_CONNECTION" value="testing"/>
```

And see if that gets the tests running?

**Comment by jbaron-mx** [comment]
Well it did something, but still failed, I've got a different error message though.

Output:
```
➜  candy git:(master) ✗ ./vendor/bin/phpunit
PHPUnit 8.5.5 by Sebastian Bergmann and contributors.

PHP Fatal error:  Cannot use GetCandy\Api\Http\Resources\Assets\AssetResource as AssetResource because the name is already in use in /home/vagrant/Code/candy/src/Http/Resources/Categories/CategoryResource.php on line 8

In CategoryResource.php line 8:

  Cannot use GetCandy\Api\Http\Resources\Assets\AssetResource as AssetResource because the name is already in use



Fatal error: Cannot use GetCandy\Api\Http\Resources\Assets\AssetResource as AssetResource because the name is already in use in /home/vagrant/Code/candy/src/Http/Resources/Categories/CategoryResource.php on line 8
```

**Comment by alecritson** [comment]
@jbaron-mx Which branch are you running on? Can you try either `master` or `develop`?

**Comment by jbaron-mx** [comment]
@alecritson Thanks man, I've got it working in `develop`! But I had to merge `master` into `develop` first for it to work because there was a fix in `master` for a dupe line 19b5756 that is missing in `develop`. I submitted PR #203 for this merge, please feel free to review the changes.
<img width="670" alt="Screen Shot 2020-05-26 at 17 41 16" src="https://user-images.githubusercontent.com/11150471/82957225-20003d80-9f78-11ea-9119-6d7b5b13eff2.png">

Thanks again for looking into this issue.

**Comment by jbaron-mx** [comment]
Closing issue. Tests have been fixed.
